configs/cms-xcache/config.d/50-cms-xcache-authz.cfg

```
@@ -1,13 +1,30 @@
 #
-# Configure security for cache servers of the CMS data federation
+# Configure authorization for the StashCache cache server.
 #
 # **********************************************************************
 # * WARNING: DO NOT EDIT THIS FILE.  IT WILL BE OVERWRITTEN ON UPGRADE *
 # **********************************************************************
 #
-# This file is part of the CMS XCache Daemon
+# This file is part of the CMS XCache Daemon                                                                                                                     
 
-# Allow connections from all hosts
-xrd.allow host *
-sec.protocol  host
-sec.protbind  * none
+xrd.allow host * 
+
+# Enable the gsi authentication using lcmaps
+
+sec.protocol /usr/lib64 gsi \
+  -certdir:/etc/grid-security/certificates \
+  -cert:/etc/grid-security/xrd/xrdcert.pem \
+  -key:/etc/grid-security/xrd/xrdkey.pem \
+  -crl:1 -authzfun:libXrdLcmaps.so \
+  -authzfunparms:--lcmapscfg,/etc/xrootd/lcmaps-cms-xcache.cfg,\
```

Use the new authzfunparms syntax:

```suggestion
  -authzfunparms:--lcmapscfg=/etc/xrootd/lcmaps-cms-xcache.cfg,\
```